reactDashboard/src/components/GeoHeatMap/GeoHeatMap.css

```diff
@@ -59,6 +68,14 @@ div.leaflet-popup-content-wrapper {
   padding: 5px;
   border: 2px solid rgba(0, 0, 0, 0.2);
 }
+
+.darkmode .legend {
+  background-color: rgb(64, 64, 64);
+  color: #f2f2f2;
+  border: 1px solid #b3b3b3;
+  border-radius: 2%;
```

Comment:
Why is border-radius darkmode specific?

Reply:
✅

Reply:
aesthetics mainly - borders are used more often in the dark mode theme to define components against the background colours i.e. black on black.  the softer corners looked nice in dark theme but didn't look good/add anything when applied to the light theme

Reply:
Can you make that clear in a css comment please?

Reply:
added comment